mmv1/products/datastream/terraform.yaml

```diff
@@ -93,6 +93,72 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       constants: templates/terraform/constants/private_connection.go.erb
       post_create: templates/terraform/post_create/private_connection.go.erb
       post_import: templates/terraform/post_import/private_connection.go.erb
+  Stream: !ruby/object:Overrides::Terraform::ResourceOverride
+    id_format: projects/{{project}}/locations/{{location}}/streams/{{stream_id}}
+    import_format: ["projects/{{project}}/locations/{{location}}/streams/{{stream_id}}"]
+    properties:
+      destinationConfig.gcsDestinationConfig.fileRotationInterval: !ruby/object:Overrides::Terraform::PropertyOverride
+        default_from_api: true
+      destinationConfig.destinationConnectionProfile: !ruby/object:Overrides::Terraform::PropertyOverride
```

Comment:
Looks like we have `projectNumberDiffSuppress`

Reply:
💯 thanks!

Reply:
added in b81ca7da9